gcs/src/components/dashboard/tabsSectionTabs/cameraTabsSection.jsx

```diff
@@ -0,0 +1,46 @@
+// NATIVE
```

Comment:
same as before, doc string at top of file and no need for caps in import comments